rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -329,6 +336,50 @@ def _decode_repay(
 
         return return_event, repay_event
 
+    def _maybe_decode_mint_events(self, context: DecoderContext) -> None:
+        """Decode possible mint events in transactions. Those are result of the aWETH
+        contract minting and transfering before withdrawing ETH. Code at
+        https://scrollscan.com/address/0xFF75A4B698E3Ec95E608ac0f22A03B8368E05F5D#code
+
+        Example tx: https://scrollscan.com/tx/0x65cd06fd54a10052c3d9084d14d28c06e2bb328b1ec39730fab9284cb529d068
+        """
+        mint = None
+        for tx_log in context.all_logs:
+            if tx_log.log_index >= context.tx_log.log_index:
+                break  # is possible to not have mint events. Also don't query after the pool event
```

Comment:
you did not do this from the previous review

```suggestion
                break  # it is possible to not have mint events. Also don't query after the pool event
```

Reply:
mmm was sure to have done it tbh